3rdparty/snmalloc/allocator.cpp

```diff
@@ -29,16 +30,10 @@ void* ThreadAllocUntyped::get()
 // see pal_open_enclave.h for details
 #include "./snmalloc/src/override/malloc.cc"
```

Comment:
```suggestion
#include "override/malloc.cc2
```
It is now in the include path.

Reply:
I will try to see if I can use malloc.cc2 in another PR. I don't think it is hight priority.